app/config/collections/projects.php

```diff
@@ -1528,7 +1528,18 @@
                 'default' => false,
                 'array' => false,
                 'filters' => [],
-            ]
+            ],
+            [
+                '$id' => ID::custom('screenshot'), // File ID from 'screenshots' Console bucket
```

Comment:
update deletes worker to remove screnshot files with build/deployments/sites/proejcts/organizations